src/main/webapp/WEB-INF/tags/admin/activity/filterPanel.tag

```diff
@@ -9,216 +9,216 @@
+                <div class="table-responsive">
+                  <table class="table table-condensed">
+                    <tr>
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Remind Students About Evaluation</li>
+                          <li class="list-group-item">Send Evaluation Closing reminders</li>
+                          <li class="list-group-item">Send Evaluation Opening reminders</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Publish Evaluation</li>
+                          <li class="list-group-item">Unpublish Evaluation</li>
+                          <li class="list-group-item">Send Registration</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Pageload</li>
+                          <li class="list-group-item">System Error Report</li>
+                          <li class="list-group-item">Servlet Action Failure</li>
+                        </ul>
+                      </td>
+                    </tr>
+
+                    <tr>
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Add New Course</li>
+                          <li class="list-group-item">Delete Course</li>
+                          <li class="list-group-item">Edit Course Info</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Enroll Students</li>
+                          <li class="list-group-item">Edit Student Details</li>
+                          <li class="list-group-item">Delete Student</li>
+                          <li class="list-group-item">Student Joining Course</li>
+                        </ul>
+                      </td>
+
+                      <td>
+                        <ul class="list-group">
+                          <li class="list-group-item">Create New Evaluation</li>
+                          <li class="list-group-item">Edit Evaluation Info</li>
+                          <li class="list-group-item">Delete Evaluation</li>
+                          <li class="list-group-item">Edit Submission</li>
+                        </ul>
+                      </td>
+                    </tr>
+                  </table>
+                </div>
+              </p>
+            </div>
+          </div>
+        </small>
+      </div>
+
+    </div>
+
+    <input type="hidden" name="pageChange" value="true">
+
+    <%-- This parameter determines whether the logs with requests contained in "excludedLogRequestURIs"
```

Comment:
If I am not mistaken, the convention for multiline JSP comment is:
```jsp
<%--
  - This is the first line.
  - This is the second line.
  --%>
```
Update this and all other multiline comments below (4 instances) to follow that convention.